tests/lib/ocpp/v201/test_charge_point.cpp

```diff
@@ -622,4 +711,49 @@ TEST_F(ChargePointFixture, K01FR22_SetChargingProfileRequest_RejectsChargingStat
     charge_point->handle_message(set_charging_profile_req);
 }
 
+/// This test ensures that when Smart Charging isn't available it doesn't process the submitted profile.
+///
+/// This test is disabled because it only passes if the ControllerComponentVariables::SmartChargingCtrlrAvailable
+/// entry is removed from the config/v201/config.json file.
```

Comment:
what is the plan to re-enable it? If it will never be enabled, it might as well be removed to avoid bitrotting.

Reply:
Alternatively you could control this variable in your test case setup like this:

```
const auto cv = ControllerComponentVariables::SmartChargingCtrlrAvailable;
this->device_model->set_value(cv.component, cv.variable.value(), AttributeEnum::Actual, "false", "TEST", true);
```

Reply:
Addressed.